client/my-sites/media-library/style.scss

```diff
@@ -5,6 +5,10 @@
 @import 'upload-button';
 @import 'upload-url';
 
+.main.media {
```

Comment:
This should probably be in `client/my-sites/media/style.scss`, since it's not specific to the modal.